bin/ch-run.c

```diff
@@ -181,9 +183,8 @@ int main(int argc, char * argv[])
       setup_namespaces(args.container_uid, args.container_gid);
       enter_udss(args.newroot, args.writable, args.binds,
                  args.private_tmp, args.private_home);
-   } else {
-      //join_namespaces();
-   }
+   } else
+      join_namespaces();
    if (args.join)
```

Comment:
Is a ```}``` missing here? You deleted one and I do not see one added anywhere. The indentation on L188 makes it look like deleting the ```}``` was unintentional.

Reply:
The GitHub diff is hiding the matching also-removed brace (line 184 in the original).